server/gateways/cloudflare.js

```diff
@@ -0,0 +1,55 @@
+import config from 'config';
+import cloudflare from 'cloudflare';
+import { get } from 'lodash';
+
+import logger from '../lib/logger';
+
+// Load config
+const cfConfig = {
+  email: get(config, 'clouflare.email'),
+  key: get(config, 'clouflare.key'),
+};
```

Comment:
Looks unnecessary compared to `config.cloudflare`.

Reply:
This object is to get passed to `cloudflare(...)` (the lib) and it only expect an `email` and a `key`. If we pass the full object it will also include the `zone` parameter.

However I understand that this can be confusing, I'll refactor